packages/server/src/workflows/thread.js

```diff
@@ -2,18 +2,61 @@ const mustache = require("mustache")
 const actions = require("./actions")
 const logic = require("./logic")
 
+const FILTER_STEP_ID = logic.BUILTIN_DEFINITIONS.FILTER.stepId
+
+function cleanMustache(string) {
+  let charToReplace = {
+    "[": ".",
+    "]": "",
+  }
+  let regex = new RegExp(/{{[^}}]*}}/g)
+  let match
+  while ((match = regex.exec(string)) !== null) {
```

Comment:
Possibly a good use case for the ES2020 `matchAll`?

Reply:
Actually blanked on this one, thanks thats perfect!